ecs-cli/modules/cli/compose/entity/task/task.go

```diff
@@ -382,6 +383,14 @@ func (t *Task) buildRunTaskInput(taskDefinition string, count int, overrides map
 		}
 	}
 
+	// TODO: revert to "LATEST" when latest refers to 1.4.0
+	if launchType == config.LaunchTypeFargate && ecsParams != nil && len(ecsParams.TaskDefinition.EFSVolumes) != 0 {
```

Comment:
nit: can we make the last condition `... > 0` to be consistent with the service check